vic/drivers/shared_image/src/vic_init.c

```diff
@@ -588,16 +588,19 @@ vic_init(void)
             }
         }
 
-        // bulk_dens_org: organic bulk density for each soil layer
-        for (j = 0; j < options.Nlayer; j++) {
-            d3start[0] = j;
-            get_scatter_nc_field_double(&(filenames.params), "bulk_density_org",
-                                        d3start, d3count, dvar);
-            for (i = 0; i < local_domain.ncells_active; i++) {
-                soil_con[i].bulk_dens_org[j] = (double) dvar[i];
+        if (!options.BULK_DENSITY_COMB) {
+            // read in bulk_dens_org:
+            // organic bulk density for each soil layer
+            for (j = 0; j < options.Nlayer; j++) {
+                d3start[0] = j;
+                get_scatter_nc_field_double(&(filenames.params),
+                                            "bulk_density_org",
+                                            d3start, d3count, dvar);
+                for (i = 0; i < local_domain.ncells_active; i++) {
+                    soil_con[i].bulk_dens_org[j] = (double) dvar[i];
+                }
```

Comment:
what happens if `BULK_DENSITY_COMB == True`? Are you sure we're parsing `BULK_DENSITY_COMB` and `ORGANIC_FRACT` properly from the global parameter file?

Reply:
@jhamman yes. If `BULK_DENSITY_COMB == True`, then we read in the combined bulk density (mineral and organic) from the parameter file rather than reading in the organic bulk density separately. 

What I don't know is to what extent reading in the combined bulk density changes soil temperatures vis-a-vis turning on the `ORGANIC_FRACT` option, but I could test that and I think it would be useful to understand. Reading in the combined bulk density *without* turning on `ORGANIC_FRACT definitely impacts soil temperatures. As @bartnijssen and I have discussed, the bulk density is used throughout the frozen soils routine, so just using the combined bulk density without turning on `ORGANIC_FRACT` is already incorporating the organic content of the soil into the thermal conductivity calculations.

Reply:
Would be good to see a test. Can be done for a single grid cell.